frontend/src/components/Gym/GymConfirmationModal.tsx

```diff
@@ -5,7 +5,7 @@ import { ButtonTypes } from '../../store/gym/types'
 import { useDispatch } from 'react-redux'
 
 function GymConfirmationModal(props: {
-  gymIsOpen: boolean
+  gymIsOpen: boolean | null
```

Comment:
should this be allowed? 
Or could we default it to false? 
Code should try to mimic real life behavior as closely as possible.

Reply:
edit: just saw the other thread oop